prow/cmd/peribolos/BUILD.bazel

```diff
@@ -49,3 +50,79 @@ go_test(
         "//vendor/k8s.io/apimachinery/pkg/util/sets:go_default_library",
     ],
 )
+
+# Usage:
+# bazel run :e2e-dump -- --github-token-path ~/whatever # --dump=whatever etc
+go_binary(
+    name = "e2e",
+    args = [
+        "--tokens=0",
+    ],
+    data = [":test-config.yaml"],
+    embed = [":go_default_library"],
+)
+
+go_binary(
+    name = "e2e-dump",
+    args = [
+        "--tokens=0",
+        "--dump=fejtaverse",
```

Comment:
looks like this is passed along to the go_image, is that desirable in this case?

Reply:
they will become CMD in the image, SGTM
https://github.com/bazelbuild/rules_docker/blob/26e5ff9a6dc292452de518e2393afe6b80f2d390/lang/image.bzl#L234

Reply:
This is a `go_binary()` target so it runs locally. 

The `go_image()` has `image = ":peribolos"` meaning it will add the binary defined on L20.